src/view.js

```diff
@@ -97,10 +97,15 @@ const View = Backbone.View.extend({
   },
 
   _getImmediateChildren() {
-    return _.chain(this._getRegions())
-      .map('currentView')
-      .compact()
-      .value();
+    return _.reduce(this._regions, this._childReducer, []);
+  },
+
+  _childReducer(children, region) {
```

Comment:
I would make `_childReducer` an standalone function instead of a method. There are two advantages: minimizes better and does not clutter view class namespace. Otherwise all good

Reply:
I moved it.  I don't love the distance it puts between the methods, but agreed that it's better in production.  There's probably some other instances where anonymous functions or private methods that don't use `this` could be extracted.